src/librustc_codegen_ironox/type_.rs

```diff
@@ -158,11 +180,23 @@ impl BaseTypeMethods<'tcx> for CodegenCx<'ll, 'tcx> {
         els: &[Type],
         packed: bool
     ) -> Type {
-        unimplemented!("type_struct");
+        let mut borrowed_types = self.types.borrow_mut();
+        let mut members = els.to_vec();
+        let struct_type = LLType::StructType {
+            name: None,
```

Comment:
I guess I was expecting a named struct version too?

Reply:
There is one: https://github.com/gabi-250/rust/blob/508c80943387801b1f3d2701b8562652d3f6ab1e/src/librustc_codegen_ironox/type_.rs#L65

Reply:
OK